ComponentKit/Accessibility/CKComponentAccessibility.h

```diff
@@ -43,17 +43,16 @@ struct CKComponentAccessibilityTextAttribute {
 
 /**
  Separate structure to handle accessibility as we want the components infrastructure to decide wether to use it or not depending if accessibility is enabled or not.
+ Not to be confused with accessibilityIdentifier which is used for automation to identify elements on the screen.
```

Comment:
Good idea adding this comment! It may be a good idea to mention that `accessibilityIdentifier` should be set on the view instead of the accessibility context.